README.md

```diff
@@ -15,6 +15,9 @@ The **Class Diagram** of the project is also provided along with the **Database
 ## Class Diagram
 ![class diagram](../master/images/diagram.PNG)
 
+**Note**: After Refactoring, new Class "HoldRequestOperations" is added to the above structure which lies in between the HoldRequest class and Book class.
+This class removes the bidirectional dependency between HoldRequest and Book. 
+
```

Comment:
@guryashdhall can we add why we made this new class and the context for this decision. Will be helpful for the readers.

Reply:
@harismuneer  I have updated the reasoning behind the introduction of new class in the README